contents/handbook/company/employee-expectations.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Expectations for PostHog employees
+sidebar: Handbook
+showTitle: true
+---
+
+## Introduction
+
+People who work at PostHog have come from all sorts of different backgrounds - large companies, small companies, agencies, single-founder startups. The modes of working at these places necessarily differ from each other and from PostHog. However, when you're brought on to work as an employee at PostHog, we expect you to work in some fairly specific ways. And being the transparent company that we are, we want to let you know about those expectations, because you can only meet expectations when you are aware of them.
```

Comment:
```suggestion
People who work at PostHog have come from all sorts of different backgrounds - large companies, small companies, agencies, single-founder startups. The modes of working at these places necessarily differ from each other and from PostHog. However, when you join PostHog, we expect you to work in some fairly specific ways. And being the transparent company that we are, we want to let you know about those expectations, because you can only meet expectations when you are aware of them.
```